setup.py

```diff
@@ -41,7 +39,5 @@
         ],
     },
     # *Minimum* requirements
-    install_requires=['numpy', 'netCDF4'],
-    ext_modules=cythonize(["src/cablab/*.pyx"]),
-    include_dirs=[numpy.get_include()]
+    install_requires=['numpy', 'netCDF4', 'gridtools'],
```

Comment:
I could not find this *gridtools* package. Do you have any link where to get it and where the docs are?